build.sc

```diff
@@ -376,6 +377,27 @@ object contrib extends MillModule {
     )
     def testArgs = T(scalanativelib.testArgs())
   }
+
+  object bsp extends MillModule {
+    // override def publishVersion = "0.0.1"
+
+    def moduleDeps = Seq(scalalib, scalajslib, main, scalanativelib)
+    def ivyDeps = Agg(
+      ivy"ch.epfl.scala:bsp4j:2.0.0-M4",
+      ivy"com.typesafe.play::play-json:2.6.9",
+      ivy"org.scala-sbt:test-interface:1.0",
+      ivy"org.scala-sbt::zinc:1.2.5",
```

Comment:
A lot of these are pretty heavy dependencies, even for an contrib module. Could we move them into a `worker` module that runs in an isolated classloader, the same way we handle `scalalib`/`scalaworker`, `scalajslib`/`scalajsworker`, etc.? That will help keep the `build.sc` classpath clean, and avoid conflicts if someone else in future also decides to take the same shortcut but with different versions of these dependencies

Reply:
Can you please provide some reference regarding where to look for some examples. Specifically the point at which scalalib.worker is invoked with the dependencies loaded in an isolated classloader

Reply:
I could get rid of those dependencies, so the above comment is no longer valid